sources/packages/backend/libs/integrations/src/services/disbursement-schedule/disbursement-schedule.models.ts

```diff
@@ -127,6 +129,21 @@ export const FULL_TIME_DISBURSEMENT_FEEDBACK_ERRORS = [
   "EDU-00112",
 ];
 
+/**
+ * Represents students Disability status both from application submitted
+ * and the student profile disability status verification.
+ */
+export interface DisabilityDetails {
+  /**
+   * Calculated PD/PPD status applied in the application by the student.
+   */
+  calculatedPDPPDStatus: boolean;
+  /**
+   * Student profile disability status.
+   */
+  studentDisabilityStatus: DisabilityStatus;
```

Comment:
I think that it would be better to be called `studentProfileDisabilityStatus` just to be more specific.